src/Domain.LinnApps/Reports/IPrefSupReceiptsReportService.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Linn.Purchasing.Domain.LinnApps.Reports
+{
+    using System;
+
+    using Linn.Common.Reporting.Models;
+
+    public interface IPrefSupReceiptsReportService
+    {
+        ResultsModel GetReport(DateTime fromDate, DateTime toDate);
+    }
+}
```

Comment:
there's quite a few of these red sign thingies for missing end of file line breaks, not sure if we can get stylecop to complain about them in visual studio? I think I've managed to get a code format command to add them on mine if I remember to run that